.github/workflows/main.yml

```diff
@@ -363,11 +363,11 @@ jobs:
     - run: |
         cargo build -p wasm-bindgen-cli
         ln -snf `pwd`/target/debug/wasm-bindgen $(dirname `which cargo`)/wasm-bindgen
-    - run: mv _package.json package.json && npm install && rm package.json
+    - run: mv _package.json package.json && corepack pnpm install && rm package.json
     - run: |
         function build() {
           (cd examples/$1 &&
-            (npm run build -- --output-path ../../exbuild/$1 ||
+            (corepack pnpm run build -- --output-path ../../exbuild/$1 ||
```

Comment:
`pnpm` resolves and installs generally much faster than `npm`.

Reply:
In general, introducing any new dependencies have to be done in their own dedicated PRs.

Reply:
It's not a new dependency, it ships with node (as does `yarn`) pre-installed.